lib/manager/composer/types.ts

```diff
@@ -8,6 +8,11 @@ export interface Repo {
 }
 export interface ComposerConfig {
   type?: string;
+  config?: {
```

Comment:
Please add tsdoc with a link to the composer docs.

Reply:
done

Reply:
> done

https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#resolve-review-comments-instead-of-commenting

😉